packages/sampleProp.conf

```diff
@@ -33,6 +33,7 @@ broker_username = "Username"
 broker_password = "Password"
 cert_required = "CERT_REQUIRED"
 tls_version = "PROTOCOL_TLSv1_2"
+crl_path = "/etc/liota/mqtt/sha2-server-g5.crl.pem"
```

Comment:
If we're going to do a single CRL file in pem format we're going to need to document how to maintain it; there's a good chance users will need to combine CRLs from multiple CAs. /etc is often read-only; we'd be better served using /usr/lib/... or wherever as our example since this the CRL is something that will need to be updated at runtime.

Reply:
I just provided an example for the user code it is not a fixed path. If we are going to combine CRL`s from multiple path then it should be placed in /usr/lib.

Reply:
When we provide examples/samples we should try to keep them realistic, so using /etc rather than /usr for the CRL is not a great example.

Generally a sysadmin will get all the CRLs they care about, combine them and then distribute them out to the machines that need them as a single file.

Reply:
I have updated the PR now the crl_path is configured using liota.conf.